FreeRTOS/FreeRTOS-Kernel
# Add a guard around mpu_wrappers.c

Description
-----------
This prevents linker errors when this file is accidentally compiled in projects using non-MPU ports.

Test Steps
-----------
Tested compiling a both and MPU and non-MPU project.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
